src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +66,24 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+
+    /**
+     * Generates a random alphanumeric string with default length 10.
+     * Credit to: https://www.baeldung.com/java-random-string
+     */
+    public static String generateRandomString() {
+        final int leftLimit = 97; // letter 'a'
+        final int rightLimit = 122; // letter 'z'
```

Comment:
possible make the final fields all capitalised?

Reply:
Cannot, constant should only be placed in the property field of the class, otherwise won't pass checkstyle.